sdk/tables/azure-data-tables/pom.xml

```diff
@@ -42,6 +43,55 @@ Licensed under the MIT License.
       <artifactId>azure-core</artifactId>
       <version>1.5.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
This doesn't look correct. The artifact version should be 1.5.1.
Can your rebase your changes onto the latest master?